src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -136,5 +142,16 @@ private bool ShouldExecuteVerification(VerifyArgs args, Verification v)
         {
             return args.Verifications.Any(verification => verification == Verification.All || verification == v);
         }
+
+        private bool IsSignatureVerifyCommandSupported()
+        {
+#if IS_DESKTOP
```

Comment:
Why don't you use `IS_SIGNING_SUPPORTED` here?

Reply:
I think there is no difference for the two?
`IS_SIGNING_SUPPORTED` is net472 + net5.0 for now,
`IS_DESKTOP` is net472.
While `RuntimeEnvironmentHelper.IsMono` only returns true when the TFM is net472.

Reply:
Does net472 works under MAC? I never tried though.

Reply:
Yes. Only in Mono. Mono is an open source implementation of Microsoft's .NET Framework.
If you check the tests running on Mac, you will see the NuGet.CommandLine.Test is able to run on Mac (should also be able to run on Linux but we just didn't). https://github.com/NuGet/NuGet.Client/blob/dev/scripts/funcTests/runFuncTests.sh#L207-L215